ivy/functional/backends/tensorflow/statistical.py

```diff
@@ -12,17 +12,44 @@
 # -------------------#
 
 
-@with_unsupported_dtypes({"2.14.0 and below": ("complex",)}, backend_version)
+@with_unsupported_dtypes(
+    {"2.14.0 and below": ("complex", "bool", "uint64")}, backend_version
+)
 def min(
     x: Union[tf.Tensor, tf.Variable],
     /,
     *,
     axis: Optional[Union[int, Sequence[int]]] = None,
     keepdims: bool = False,
+    initial: Optional[Union[int, float, complex]] = None,
+    where: Optional[Union[tf.Tensor, tf.Variable]] = None,
     out: Optional[Union[tf.Tensor, tf.Variable]] = None,
 ) -> Union[tf.Tensor, tf.Variable]:
     axis = tuple(axis) if isinstance(axis, list) else axis
-    return tf.math.reduce_min(x, axis=axis, keepdims=keepdims)
+    if where is not None:
+        if x.dtype == tf.int8:
```

Comment:
I think we could simplify lines 30-49 with 
```
max_val = ivy.iinfo(x.dtype).max if ivy.is_int_dtype(x.dtype) else ivy.finfo(x.dtype).max
```
Same applies to the `paddle` backend implementation

Reply:
Made the changes. Thanks!

Reply:
awesome!